cmd/cmd.go

```diff
@@ -875,8 +868,27 @@ func checkServerHeartbeat(_ *cobra.Command, _ []string) error {
 	return nil
 }
 
+func versionHandler(cmd *cobra.Command, _ []string) {
+	fmt.Printf("ollama version %s\n", version.Version)
+
+	client, err := api.ClientFromEnvironment()
+	if err != nil {
+		return
+	}
+
+	serverVersion, err := client.Version(cmd.Context())
+	if err != nil {
+		return
```

Comment:
I think using the client version and not logging when the server version check fails could lead to a scenario where we are troubleshooting on the wrong version.

Ex: The server fails to start or crashes, and when a user reports the error they send the version from the CLI, which may not be accurate due to the server not being responsive.